src/styles/global.css

```
@@ -88,3 +88,67 @@ span[id="footnote-label"] {
   border-bottom: 1px solid theme(colors.muted.foreground / 40%);
   font-weight: 700;
 }
+
+@font-face {
+  font-family: "LINE Seed JP";
+  src: url("/fonts/line-seed-jp/LINESeedJP_OTF_Rg.woff2") format("woff2");
+  font-weight: regular;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: "LINE Seed JP";
+  src: url("/fonts/line-seed-jp/LINESeedJP_OTF_Bd.woff2") format("woff2");
+  font-weight: bold;
```

```suggestion
  font-weight: theme(fontWeight.bold);
```

些細なことですが、他の個所でtheme関数を使っているので一応